### PR TITLE
Add new content to contact page

### PR DESCRIPTION
**Pull request by hectoruch**
Task => https://www.pivotaltracker.com/story/show/130174377

### CODE REVIEW

File: app/src/components/ContactUs/ContactUsForm.jsx [hunk 1]
```diff
@@ -173,6 +173,18 @@ class ContactUsForm extends Component {
             className={buttonStyle['c-button-contact']}
           >SEND</button>
         </form>
+        <div className={contactStyle['emails-text']}>
+          For media or press inquiries, please contact:
+          <ul>
+            <li>Oceana: Dustin Cranor, <a href="mailto:youremailaddress">dcranor@oceana.org</a>, 954.348.1314</li>
```
Comment (clementprdhomme) [inline app/src/components/ContactUs/ContactUsForm.jsx]: `youremailaddress`? 😂

Don't forget the two other below.

File: app/src/components/ContactUs/ContactUsForm.jsx [hunk 2]
```diff
@@ -173,6 +173,18 @@ class ContactUsForm extends Component {
...
+            <li>SkyTruth: Jenny Allen, <a href="mailto:youremailaddress">jenny@skytruth.org</a>, 304.582.2031</li>
+            <li>Google: Mara Harris, <a href="mailto:youremailaddress">press@google.com</a></li>
+          </ul>
+        </div>
+        <div className={contactStyle['emails-text']}>
+          For Research Program related inquiries, please contact
```
Comment (clementprdhomme) [inline app/src/components/ContactUs/ContactUsForm.jsx]: Please add `{' '}` at the end of the line because the email address is stuck to "contact".

**Comment by clementprdhomme** [comment]
I added two comments. Can you make the changes?

File: app/src/components/ContactUs/ContactUsForm.jsx (hunk 1 above)
Comment (hectoruch) [inline app/src/components/ContactUs/ContactUsForm.jsx]: Oh my god...sorry for that!

**Comment by hectoruch** [comment]
I just fix all @clementprdhomme

**Pull request merged**